liberty-maven-plugin/src/test/resources/checkErrorVersionlessFeature.bsh

```diff
@@ -9,6 +9,9 @@ switch (testname) {
     case "kernel-install-versionless-feature-fail-no-platform-it":
         expectedMessage = "CWWKF1516E: The platform could not be determined. The following versionless features cannot be installed: [servlet]";
          break;
+    case "springboot-3-appsdirectory-apps-fail-it":
```

Comment:
So I thought I made this point earlier, but if we are going to have a check in this file that is not related to versionless features then the file should be renamed (it is currently checkErrorVersionlessFeature.bsh). Alternatively, you could have a separate bsh file for checking this error related to springBootApplication.

Reply:
Moved to a new file 

![Uploading image.png…]()

Reply:
You should have removed your changes to this file when you added the check to the new file.